File: back/src/api/models/application.py
from pydantic import BaseModel, Field

class ApplicationSchema(BaseModel):
    application_id: int     =   Field(default=None)
    name:           str     =   Field(...)
    status:         str     =   Field(...)
    claim:          int     =   Field(...)
    description:    str     =   Field(...)

    class Config:
        schema_extra = {
            "example": {
                'name': 'testname',
                'status': 'на диагностике',
                'claim': 0,
                'description': 'testdescription'
            }
        }

    def to_dict(self) -> dict:
        return {
            'name':         self.name,
            'status':       self.status,
            'claim':        self.claim,
            'description':  self.description
        }

class UserApplicationRelaionSchema(BaseModel):
    application_id: int = Field(...)
    user_id: int = Field(...)
    user_role: str = Field(...)

    def to_dict(self) -> dict:
        return {
            'case_id': self.application_id,
            'user_id': self.user_id,
            'user_role': self.user_role
        }

File: front/SudInvest/src/app/models/application.ts
export class Application {
    application_id: number = 0;
    initiator_id: number = 0;
    name: string = "";
    status: string = "";
    claim: number = 0;
    description: string = "";
    initiator_role: string = "";

    constructor(
        application_id: number,
        initiator_id: number,
        name: string,
        status: string,
        claim: number,
        initiator_role: string,
        description: string)
    {
        this.application_id = application_id;
        this.initiator_id = initiator_id;
        this.name = name;
        this.status = status;
        this.claim = claim;
        this.initiator_role = initiator_role;
        this.description = description;
    }
}